components/benefit_sponsors/app/views/benefit_sponsors/benefit_packages/benefit_packages/estimated_employee_cost_details.html.erb

```diff
@@ -103,9 +103,13 @@
   </div>
 </div>
 
-<%= link_to 'Return to Plan Selection', edit_benefit_sponsorship_benefit_application_benefit_package_path(@benefit_sponsorship, @benefit_application, @benefit_package, :show_benefit_application_tile => false), method: :get, class: 'btn btn-default pull-right mb-1' %>
+<%= link_to 'Return to Plan Selection', edit_benefit_sponsorship_benefit_application_benefit_package_path(@benefit_sponsorship, @benefit_application, @benefit_package, :show_benefit_application_tile => false), method: :get, class: 'btn btn-default btn-pull-right pull-right mb-1' %>
```

Comment:
Pull-right class is define to send element to right side. So that’s the reason I removed and added a new class to make it left. Yeah, I will look into it.

Reply:
I have updated the code, please review